optuna/study.py

```diff
@@ -44,6 +44,9 @@
     ObjectiveFuncType = Callable[[trial_module.Trial], float]
 
 
+logger = logging.get_logger(__name__)
```

Comment:
How about marking the object private by prefixing it with an underscore, as it's not very likely to be used from outside this submodule?

Reply:
makes sense.

I didn't know that optuna/integration/sklearn.py does that.

Reply:
Me neither, until you mentioned. 😅